cosmoz-default-tree.html

```diff
@@ -161,29 +165,19 @@
 
 		var path = pathLocator.split(pathLocatorSeparator),
 			pathSegment = nodeObj || this._treeData,
-			nodes = [];
+			nodes = [],
+			node;
 
-		// Get the nodes on the path.
-		var validPath = path.every(function (nodeKey) {
-			if (!pathSegment || !pathSegment[nodeKey]) {
-				return nodes.length === 0;
-			}
-			var node = pathSegment[nodeKey];
-			pathSegment = node[this.childProperty];
+		// Get the nodes on the path
+		path.forEach(function (nodeKey) {
```

Comment:
This implementation looks very nice, I would just like to see it as a `map()` again now since the array push is done on each iteration anyway. Also we need to make sure it handles the scenario I commented about in the method docs.